### PR TITLE
cmake build type as param

### PR DESCRIPTION
**Pull request by mathieu-aubin**
substitution to make cmake runs more alike.
Also, question... why have this command multiline while others are one liners?
Why not make them the same 'format' (either way, but the same)?

Feel free to ignore/close this if not relevant

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -299,8 +299,9 @@ RUN \
     -DBUILD_STATIC_LIBS=ON \
     -DCMAKE_INSTALL_PREFIX=/usr/local \
     -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_BUILD_TYPE=Release \
     .. \
-  && cmake --build . --config Release --target install
+  && cmake --build --target install .
```
Comment (wader) [inline Dockerfile:304]: Build fail with `#24 1.031 Unknown argument install` wonder if `--build` still wants the "."?

**Comment by wader** [comment]
> substitution to make cmake runs more alike. Also, question... why have this command multiline while others are one liners? Why not make them the same 'format' (either way, but the same)?

There's no good reason why they differ so feel free to reformat the other ones also if you want. Make them all multiline maybe? easier to read i think

File: Dockerfile (hunk 1 above)
Comment (mathieu-aubin) [reply Dockerfile:304]: i might have messed up here with this. it seems obvious now. As this is absolutely not important, i will close

**Comment by mathieu-aubin** [comment]
agree with the multiline

File: Dockerfile (hunk 1 above)
Comment (wader) [reply Dockerfile:304]: no worries! will probably cleanup the cmake stuff at some point if you don't beat me to it

**Pull request closed without merging**